ngraph_bridge/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ set(SRC
    ngraph_encapsulate_impl.cc
    ops/ngraph_ops.cc
    ngraph_encapsulate_op.cc
+   ngraph_executor_db.cc
```

Comment:
Should add header and .cc file for bazel here - https://github.com/tensorflow/ngraph-bridge/blob/master/bazel/BUILD#L22